packages/loaders/src/Skeleton.js

```diff
@@ -75,9 +74,9 @@ const StyledSkeleton = styled.div.attrs({
     /* stylelint-disable */
```

Comment:
Please either fix the stylelint config to allow this exception globally or use the disable syntax that's indicating exactly why you need the exception here. Ex: `stylelint-disable function-comma-space-after */